libraries/AP_Scripting/generator/description/bindings.desc

```diff
@@ -206,6 +206,8 @@ singleton AP_Vehicle method get_control_output boolean AP_Vehicle::ControlOutput
 singleton AP_Vehicle method get_target_location boolean Location'Null
 singleton AP_Vehicle method set_target_velocity_NED boolean Vector3f
 singleton AP_Vehicle method set_target_angle_and_climbrate boolean float -180 180 float -90 90 float -360 360 float -FLT_MAX FLT_MAX boolean float -FLT_MAX FLT_MAX
+singleton AP_Vehicle method get_circle_radius boolean float'Null
+singleton AP_Vehicle method set_circle_rate boolean float -FLT_MAX FLT_MAX
```

Comment:
Don't we save bytes by omitting the limits now?

Reply:
I tried removing the limit but I received an error message regarding the lower limit missing..

Reply:
the no check stuff is not in yet, https://github.com/ArduPilot/ardupilot/pull/18221